service/raft/group0_state_machine.cc

```diff
@@ -169,6 +169,11 @@ future<> group0_state_machine::merge_and_apply(group0_state_machine_merger& merg
         co_await write_mutations_to_database(_sp, cmd.creator_addr, std::move(chng.mutations));
         co_await _ss.topology_transition();
     },
+    [&] (mixed_change& chng) -> future<> {
+        co_await _mm.merge_schema_from(netw::messaging_service::msg_addr(std::move(cmd.creator_addr)), std::move(chng.mutations));
+        co_await _ss.topology_transition();
+        co_return;
+    },
```

Comment:
BTW, This processing method is not object oriented.
I wonder if `group0_command` could be a virtual class with derived class implementing the above as a method.
In any case, it's just food for thought and not part of this PR.

Reply:
> BTW, This processing method is not object oriented.

You are saying it as if it is a bad thing. We are not in 90s.

Reply:
Not relevant to this PR, closing.

Reply:
> > BTW, This processing method is not object oriented.
> 
> You are saying it as if it is a bad thing. We are not in 90s.

Seems like we're getting back to the 80s then, just using std::visit instead of a switch on the type... ;-)
But joking aside, why not use straightforward inheritance for specializing the processing of the different command types?

Reply:
> > > BTW, This processing method is not object oriented.
> > 
> > 
> > You are saying it as if it is a bad thing. We are not in 90s.
> 
> Seems like we're getting back to the 80s then, just using std::visit instead of a switch on the type... ;-) 

What can be more object oriented that a visitor design patter?

> But joking aside, why not use straightforward inheritance for specializing the processing of the different command types?

There is nothing straightforward about inheritance.

Reply:
Nowadays functional programming™ is in vogue, and `std::visit` is the functional™ way of doing stuff